guide.md

```diff
@@ -0,0 +1,4 @@
+# Next Steps
```

Comment:
we gotta add instructions down the line for sure (not necessarily in this PR ofc). also maybe we could better maintain discussions and progress on future ideas on the github issue tracker instead of the file?

Reply:
ye ye sounds rights

Reply:
i realised i should've done a squash commit 💀 probably (definitely) was too sleepy for it